### ISSUE TITLE
student computer setup for the module

### ISSUE THREAD

**Issue by martinjrobins** [SABS-R3/software-engineering-day1#7]
I've been thinking about what the students will need for the module re software. As you know we have the SABs laptops from last year, which we will be collecting from the current students in Induction week (5-8th Oct). I'll image these on the friday (9th) and they will be available for the students to pick up first thing on the monday morning of the module (Steve, James: you might want to build that into the timetable for monday morning).

However, some students won't be physically in Oxford (e.g. we have one who is in Australia, which isn't letting ppl out of the country), so we'd need to provide other options. My thinking was that we give the students three options:

1. pick up a SABs laptop
2. download a virtual machine image which we provide and install virtualbox on your personal computer (only if you have at least 8 GB of RAM). Load up the image in Virtualbox
3. download and install Ubuntu 20.04.1 LTS on your personal computer, install software as in #8 

That should cover everyone except those who don't have an available computer, in that case we might have to consider other options (like SABs buying them a laptop). In either case we'll need to let the students know soon so we can work out any problems. Does this sound reasonable?

**Comment by martinjrobins** [comment]
list of software for the module here: #8

**Comment by fcooper8472** [comment]
Yes, this sounds reasonable to me.  Most important thing will be having everyone on as close as possible to the same software or it might end up being a nightmare to help troubleshoot, especially remotely.

**Comment by iamleeg** [comment]
If we standardise on a particular distro like ubuntu 20.04, we could provide a script that apt-gets the necessary packages to avoid students getting weird versions, or getting similarly-named but wrong packages. And the script would presumably be used to configure the SABS laptops and the VM image too :)

**Comment by smangham** [comment]
There was one missing library, I think on the student laptops - `libffi-dev`.
